utils.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/bitrise-io/go-steputils/tools"
+	"github.com/bitrise-io/go-utils/colorstring"
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/fileutil"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-io/go-utils/stringutil"
+)
+
+func printLastLinesOfXcodebuildTestLog(rawXcodebuildOutput string, isRunSuccess bool) {
+	const lastLines = "\nLast lines of the build log:"
+	if !isRunSuccess {
+		log.Errorf(lastLines)
+	} else {
+		log.Infof(lastLines)
+	}
+
+	fmt.Println(stringutil.LastNLines(rawXcodebuildOutput, 20))
+
+	if !isRunSuccess {
+		log.Warnf("If you can't find the reason of the error in the log, please check the xcodebuild_test.log.")
+	}
+
+	log.Infof(colorstring.Magenta(`
```

Comment:
The message does not match with the name of the actual constant (BITRISE_XCODE_RAW_RESULT_TEXT_PATH); I recall there was a dsicussion in the earlier Inptus revamp to not cahnge that, but do not remember the exact reasons.

Reply:
Kept the original env var (`BITRISE_XCODE_RAW_RESULT_TEXT_PATH`) and updated these logs to print the truth.